content/en/plugins/github-api/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: GiHub API Plugin for Spinnaker
+linkTitle: GitHub API
+no_list: true
+description: >
+  Armory's GitHub API Plugin provides features that enable engineers to trigger Spinnaker pipelines from GitHub workflows, update GitHub deployment status based on pipeline outcome, and view GitHub Action logs in Spinnaker.
+---
+
+
+## GitHub API Plugin features
+
+The GitHub API Plugin provides the following GitHub integration features:
+
+- Trigger a Spinnaker pipeline from a GitHub workflow using **workflow_dispatch** or **repo_dispatch** events
+- Trigger a Spinnaker pipeline automatically when a GitHub workflow finishes successfully
+- Trigger a Spinnaker pipeline when GitHub creates a new GitHub Deployment
+- Monitor a GitHub workflow and finish pipeline execution based on the GitHub workflow result
+- Update GitHub deployment status based on Spinnaker pipeline outcome
+- View Github Action Logs in Spinnaker -- there is no need to navigate to GitHub to view the logs
+
+The GitHub API plugin uses [GitHub Apps](https://docs.github.com/en/apps/overview) to integrate with GitHub. GitHup Apps provide webhooks and narrow, specific permissions. You install a GitHub App in your organization and specify which repositories the GitHub API plugin can access. 
+
+## How the plugin works with GitHub
+
+```mermaid
+graph TD
+	Orca -->|Workflow Trigger| Igor
+	Gate --> |Trigger based on Deployment event|Echo
+	Igor -->|Trigger/Monitor triggered workflows| Github
+	Echo -->|Pipeline triggers|Orca
+	Orca -->|Pipeline notifications|Echo
+	Github --> |Deployment event| Gate
+	Echo --> |Notifications|Github
+```
+
+For example, the plugin processes a GitHub Deployment event like this:
+
+```mermaid
+sequenceDiagram
+participant ghActions as "Github Workflow job"
+participant gh as "Github"
+participant gate as "Gate"
+participant echo as "Echo"
+participant orca as "Orca"
+
+ghActions ->> gh: "Create Deployment"
+gh ->> ghActions: "Deployment created"
+gh ->> gate: "Deployment Event"
+gate ->> echo: "Deployment trigger event"
+echo ->> orca: "Trigger pipelines matching the Trigger event"
+orca ->> gh: "Deployment status update: in_progress, success, failure, or error"
+```
+
+## Compatibility matrix
+
+{{< include "plugins/github/compat-matrix.md" >}}
+
+## Installation paths
+
+{{% cardpane %}}
+{{% card header="Armory CD<br>Armory Operator" %}}
+Use a Kustomize patch to install the plugin.
+
+1. Create a GitHub App and install it in your repo.
```

Comment:
`Create a GitHub App and install it in your repo.` maybe it will be better to be replaced with `Create a GitHub App and install it in your repo or Github organization`

Reply:
I added a note to retrieve the App ID and Private key to the install instructions